parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveStringifier.java

```diff
@@ -331,6 +330,32 @@ private long convert(long duration, TimeUnit from, TimeUnit to, TimeUnit higher)
     }
   };
 
+  static PrimitiveStringifier createTimestampStringifier(final LogicalTypeAnnotation.TimeUnit timeUnit, final boolean isAdjustedToUTC) {
```

Comment:
Since others may use this code as a reference for proper timestamp-handling, I think it would better to use the correct types (`Instant` for UTC-normalized and `LocalDateTime` for non-UTC-normalized) instead of just printing the timestamps correctly.